ihm/dumper.py

```diff
@@ -1485,6 +1485,7 @@ def dump_atoms(self, system, writer):
                              label_asym_id=atom.asym_unit._id,
                              label_entity_id=atom.asym_unit.entity._id,
                              label_seq_id=atom.seq_id,
+                             auth_seq_id=atom.asym_unit.auth_seq_id_map[atom.seq_id] if isinstance(atom.asym_unit.auth_seq_id_map, dict) else atom.seq_id,
```

Comment:
1. This line is too long (`flake8` limits lines to 80 chars).
2. Would be much cleaner and simpler to use `auth_seq_id=atom.residue.auth_seq_id` here as `auth_seq_id` is a per-residue property.